web-ui/src/main/resources/catalog/locales/nl-admin.json

```diff
@@ -180,6 +180,7 @@
     "data.configDir": "Configuratie folder:",
     "data.dataDir": "Data map:",
     "data.formatterPath": "Formatter folder:",
+    "data.htmlCacheDir": "HTML cache folder:",
```

Comment:
Should not be done in Transifex?

Reply:
We will have to translate it in transifex once the PR is merged but having it here is not really a problem. It will be overwritten